Dockerfile

```diff
@@ -11,9 +11,11 @@ RUN apt-get update \
     groff \
     jq \
     less \
+    openssl \
     python3 \
     python3-pip \
-    python3-dev
+    python3-dev \
+    vim
```

Comment:
Add vim so `kubectl edit` works

Reply:
Might make sense to specify vim-nox.  I think 'vim' is a virtual package; I'm not sure which gets pulled in by default when you install it and if it's one of the -gtk variants that may make this image a lot larger than it needs to be.